### ISSUE TITLE
Instant alert screen?

### ISSUE THREAD

**Issue by poblabs** [lubeda/EsphoMaTrix#72]
Hi, is it possible to have a new method to have an instant alert screen? I have tried to add a screen, and force that screen but it didn't work immediately. I'm looking for something that will stop the current loop, display the alert, then go back to the loop.

**Comment by lubeda** [comment]
i thought this was allready fixed, i will take a look

**Comment by lubeda** [comment]
i can't find any error on this.

you have to add a screen to the queue, then you can immediatly force it to the display

**Comment by poblabs** [comment]
I guess I'm looking for something different. I do not want a screen in queue to show. I want to be able to instantly alert on something, have it pushed to the screen to show, then it can go away. If that doesn't exist, should I open a feature request?

**Comment by lubeda** [comment]
Thats already available, e.g. with this automation triggerd by incoming phone call:

```
- id: "1xyxcyxc7143"
  alias: EHMTX Anruf anzeigen
  trigger:
    - platform: state
      entity_id: sensor.fritz_box_6490_cable_lgi_call_monitor_phone
      attribute: to
      to: "0555123555"
  condition: []
  action:
    - data:
        icon_name: phone
        text: "someone is calling"
        lifetime: 1
        screen_time: 60
      service: esphome.awtrix_add_screen
    - service: esphome.awtrix_force_screen
      data:
        icon_name: phone
  mode: single
```

The icon will disapear after one minute (lifetime)

**Comment by poblabs** [comment]
I'll try it, thanks!

**Comment by lubeda** [comment]
thanks for this detailed analytics

**Comment by lubeda** [comment]
1 and 2 are fixed in fixed in 2023.4.1b

but 2023.4.1b is a complete redesign, so it is not usable yet

**Comment by poblabs** [comment]
I will try it when it's available, thanks!
